doc/getting_started.rst

```diff
@@ -0,0 +1,230 @@
+
+``Scikit-learn`` is an open source machine learning library that supports
+supervised and unsupervised learning. It also provides various tools for
+model fitting, data preprocessing, model selection and evaluation, and many
+other utilities.
+
+Fitting and predicting: estimator basics
+----------------------------------------
+
+``Scikit-learn`` provides dozens of built-in machine learning algorithms and
+models, called :term:`estimators`. Each estimator can be fitted to some data
+using its :term:`fit` method.
+
+Here is a simple example where we fit a
+:class:`~sklearn.ensemble.RandomForestClassifier` to some very basic data::
+
+  >>> from sklearn.ensemble import RandomForestClassifier
+  >>> clf = RandomForestClassifier(random_state=0)
+  >>> X = [[ 1,  2,  3],  # 2 samples, 3 features
+  ...      [11, 12, 13]]
+  >>> y = [0, 1]  # classes of each sample
+  >>> clf.fit(X, y)
+  RandomForestClassifier(random_state=0)
+
+The :term:`fit` method generally accepts 2 inputs:
+
+- The samples matrix (or design matrix) :term:`X`. The size of ``X``
+  is typically ``(n_samples, n_features)``, which means that samples are
+  represented as rows and features are represented as columns.
+- The target values :term:`y` which are real numbers for regression tasks, or
+  integers for classification (they can also be strings). For unsupervized
+  learning tasks, ``y`` does not need to be specified. ``y`` is usually 1d
+  array where the ``i`` th entry corresponds to the target of the ``i`` th
+  sample (row) of ``X``.
+
+Both ``X`` and ``y`` are usually expected to be numpy arrays or equivalent
+:term:`array-like`, data types, though some estimators work with other
+formats such as sparse matrices.
+
+Once the estimator is fitted, it can be used for predicting target values of
+new data. You don't need to re-train the estimator::
+
+  >>> clf.predict(X)  # predict classes of the training data
+  array([0, 1])
+  >>> clf.predict([[4, 5, 6], [14, 15, 16]])  # predict classes of new data
+  array([0, 1])
+
+Transformers and pre-processors
+-------------------------------
+
+Machine learning worflows are often composed of different parts. A typical
+pipeline consists of a pre-processing step that transforms or impute the
+data, and a final predictor that predicts target values.
+
+In ``scikit-learn``, pre-processors and transformers follow the same API as
+the estimator objects (they actually all inherit from the same
+``BaseEstimator`` class). The transformer objects don't have a
+:term:`predict` method but rather a :term:`transform` method that outputs a
+newly transformed sample matrix ``X``::
+
+  >>> from sklearn.preprocessing import StandardScaler
+  >>> X = [[0, 15],
+  ...      [1, -10]]
+  >>> StandardScaler().fit(X).transform(X)
+  array([[-1.,  1.],
+         [ 1., -1.]])
+
+Sometimes, you want to apply different transformations to different features:
+the :ref:`ColumnTransformer<column_transformer>` is designed for these
```

Comment:
the user guide may be a better link than the API page.

Reply:
it's the user guide :)